### PR TITLE
feat: add Tips #19

### PR DESCRIPTION
**Pull request by elrrrrrrr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,21 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter.
...
+## #18 - Node.js: Detect a module before actually requiring
+
+> 2016-01-18 by [@elrrrrrrr](https://github.com/elrrrrrrr)
+
+In node, you can detect a module before actually requiring it. Here is the simple helper function. This is useful in detecting global dependency or dynamic loading modules.
+
+```js
+try {
+  require.resolve('name'); // .resolve will not load the module if not found.
+  return true;
```
Comment (0xmtn) [inline README.md]: You `return` inside try-catch, without defining the function.

**Comment by loverajoel** [comment]
@elrrrrrrr ping

**Comment by homer0** [comment]
Just to be annoying :P, would you mind adding an example where you would want to detect the module before requiring it?

**Comment by elrrrrrrr** [comment]
@0xmtn  added function definition.

**Comment by elrrrrrrr** [comment]
@homer0   such as loading the "~/config.js" .

**Comment by homer0** [comment]
@elrrrrrrr ohh, sorry, I didn't understand it at first, by "detect", I thought you were saying that it knows if the module was loaded or not (yeah, the shame... I didn't went to the docs :P).

Please consider changing the _"you can detect a module before actually requiring it"_ to something like _"you can check if a module can be required or not"_; just a suggestion :P.

Other than the suggestion, :skull: like it!

**Pull request closed without merging**